routes/comment.py

```diff
@@ -5,6 +5,21 @@
 
 api = Api(app)
 
+class GetAllComments(Resource):
+  def get(self):
+    comments = mongo.db.comments
+    data = []
+
+    for field in comments.find():
+      data.append(
+        {
+          '_id': str(field['_id']),
+          'username': field['name'],
+          'comment': field['comment']
+        }
+      )
+    return jsonify(data)
```

Comment:
What happens if we didn't do the loop and just `jsonify(comments.find())`

Reply:
it will throw an error.

TypeError: Object of type method is not JSON serializable